data/reference/Publications.json

```diff
@@ -7468,5 +7468,29 @@
         "bibcode": "",
         "doi": "",
         "description": "Diversity of Cold Worlds: A Near Complete Spectral Energy Distribution for 2MASS J04151954-0935066 using JWST"
+    },
+    {
+        "reference": "\u017derj17",
```

Comment:
uh oh, we have a unicode problem here. Example for https://github.com/astrodbtoolkit/astrodb_utils/issues/113